### PR TITLE
minutes added for ed-wg-2022-02-16

### PR DESCRIPTION
**Pull request by jonathondilworth**
Meeting minutes for the educational working group - there was a large crossover into marketing. TLDR is at the top, minutes are detailed, actionables are clear.

### CODE REVIEW

**Comment by jonathondilworth** [comment]
_Creating a quick 'how to use' GitHub video (how to add reviewers)._ We'll likely merge this branch later today unless there are any objections.

File: Education/ED-WG-2022-02-17.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+### ED-WG Meeting Minutes 2022-02-16
+
+*Warning, this is almost a kind of transcript. A very effective hour and a half. Thus, see TLDR.*
+
+#### TLDR
+
+* Miro > GitHub.
+* Use GitHub to PR minutes & important docs into Documents.
```
Comment (thenic95) [inline Education/ED-WG-2022-02-17.md:8]: Great way to approach this.

File: Education/ED-WG-2022-02-17.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+* Begin indexing library & creating a knowledge base of sorts (easy to navigate materials for path-based learning).
```
Comment (thenic95) [inline Education/ED-WG-2022-02-17.md:9]: Is there a plan what to use for indexing? Spreadsheet, simple DB, ...

File: Education/ED-WG-2022-02-17.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+* **Contact Tommy & Allusian about indexing materials.**
+	* Owners: Dilworth, Tommy, Allusian.
+	* Lead: Dilworth.
+
+* **If we're going to use GitHub and regularly make PRs containing important documents, perhaps we should have an event (webhook) that pushes the PR for the appropriate working group chat + manually mirror to Miro (up for debate).**
+	* Group Discussion.
+	
+	
+	
+
```
Comment (thenic95) [inline Education/ED-WG-2022-02-17.md:168]: I really enjoy the TL:DR section, that helps members that couldn't attend to quickly understand what was discussed and if they should continue reading.
Well done!
It's really important to convey outcomes of those meetings with as little text as possible.

**Review by thenic95 — COMMENTED** [review]
I really enjoyed the TLDR section, this helps ADAO members that couldn't attent to quickly understand what was discussed.
Personally I wouldn't add every comment from other members, it's definetly up to you but what matters most are the outcomes.

Great minutes, well done!

File: Education/ED-WG-2022-02-17.md (hunk 2 above)
Comment (jonathondilworth) [reply Education/ED-WG-2022-02-17.md:9]: Yeah, so it looks like we're going to start with a simple spreadsheet. But, there are some concerns with overwhelming options => people are more likely to simply do nothing (outlined some research & references on the Miro board). So, what I had in mind & what will likely be discussed is:

- Option One: Simple Spreadsheet
- Option Two: Basic Search (Information Retrieval)
- Option Three: Potential for a 'Knowledge Base' (Tag materials and use slightly more complicated IR to pull in indexed materials - I've done some stuff like this before).

Then the whole path based learning is another idea, but would also require tagging materials and implementing some kind of recommender systems (interface could be through discord or a web app).

& finally, we'll likely be discussing whether or not we're going to need materials for git & GitHub education.

If you'd like to check out the work been done on this, it's all available (as of this morning) on the Miro board :)

Comment (thenic95) [reply Education/ED-WG-2022-02-17.md:9]: Thank you for your quick answer!
I'd say option three sounds best and is probably what we will end up with anyway. 
Spreadsheets can just be used in the meantime, that's how I would do it.
But I'm sure you and Bob will find a perfect solution :)

**Pull request merged**